tests/test_data.py

```diff
@@ -16,6 +16,7 @@
     available_to_mentor=True,
     need_mentoring=True
 )
+
 user2 = dict(
```

Comment:
replace with literal dict syntax

Reply:
I did it..... but causes 124 test cases to fail.... ss below is when I changed it to literal dict syntax...

![1](https://user-images.githubusercontent.com/40512667/52726866-084ab300-2fda-11e9-9637-359267753644.png)